.travis.yml

```diff
@@ -96,6 +96,30 @@ matrix:
             - llvm-toolchain-trusty-4.0
           packages:
             - clang-4.0
+    - os: linux
+      branches:
+        only:
+	  - autotools.marcomaggi-2017-05-17
+      compiler: gcc-5
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - gcc-5
+      install:
+        - ./meta/travis-ci/install-autoconf.sh
```

Comment:
You can use the trusty dist `dist; trusty` it will accelerate the build time and avoid the need of such scripts